data/projects/jetbrains-ide-plugin.md

```diff
@@ -0,0 +1,24 @@
+JetBrains is a software vendor specializing in the creation of intelligent development tools. JetBrains IDEs 
+includes IntelliJ IDEA, PyCharm, CLion, Gogland and so on.
+
+We'd like to integrate coala with jetbrains IDEs, just like 
+[coala-atom](https://github.com/coala/coala-atom) does.
+
+#### Milestones
+
+##### GSOC 2017 CODING STARTS
+
+* A concept exists for the project.
+* An official repository set up.
+
+##### GSOC 2017 MIDTERM
+
+* A working plugin with minimal functionality exists.
+* A basic testsuite exists.
+* Basic documentation exists.
```

Comment:
How about `corresponding test cases exists and corresponding documentation exists`